Makefile

```
@@ -1,7 +1,13 @@
-prepare:
-	python3.11 -m venv env
-	env/bin/python -m pip install robotframework
+venv:
+	python3.11 -m venv .venv
+	python -m pip install robotframework
+	python -m pip install requests
+	python -m pip uninstall robot
```

This will build the venv if the .venv/touchfile and the .venv directory do not exist and creates an .venv/touchfile so that if you run the `make venv` again it does not need to do anything.

```suggestion
venv: .venv/touchfile ## Create virtual environment
.venv/touchfile:
	test -d .venv || python3.11 -m venv .venv
	. .venv/bin/activate && pip install robotframework
	. .venv/bin/activate && pip install requests
	touch .venv/touchfile
```